src/main/java/lz/modelTesting/configuration/MongoDBConfig.java

```diff
@@ -35,11 +35,31 @@ CommandLineRunner loadSamples(UsersRepository usersRepository, MockupsRepository
+            surveysRepository.save(new Survey("First Sample Survey", surveyBody, basicType));
+            surveysRepository.save(new Survey("Second Sample Survey", surveyBody, basicType));
+            surveysRepository.save(new Survey("Third Sample Survey", surveyBody, basicType));
+
+            for (int i = 0; i < 5; i++) {
+                projectsRepository.save(projects[i]);
+                Mockup mockup1 = new Mockup("Axure", "https://2usnmc.axshare.com/", projects[i].getId());
+                mockupsRepository.save(mockup1);
+                Survey survey1 = new Survey("Mockup Sample Survey " + (i + 1), surveyBody, "mockup");
+                survey1.setMockupId(mockup1.getId());
+                surveysRepository.save(survey1);
+                mockupsRepository.save(new Mockup("AdobeXD", "https://xd.adobe.com/embed/e6a0d97b-6bfc-4f07-653f-70a6a2eae5a7-9091/", projects[i].getId()));
+                mockupsRepository.save(new Mockup("Figma", "https://www.figma.com/embed?embed_host=share&url=https://www.figma.com/proto/a32Lpn3oXSef2HgPtu5BQx/Course-Dashboard-Copy?node-id=1%3A10&scaling=scale-down-width", projects[i].getId()));
+            }
+
             saveSampleProject(mockupsRepository, projectsRepository);
-            Survey test_survey = new Survey("Test Survey", "{\"questions\":[{\"isRequired\":false,\"enableIf\":true,\"name\":\"Is this working?\",\"type\":\"comment\",\"title\":\"Is this working?\"}]}");
+            Survey test_survey = new Survey("Test Survey", "{\"questions\":[{\"isRequired\":false,\"enableIf\":true,\"name\":\"Is this working?\",\"type\":\"comment\",\"title\":\"Is this working?\"}]}", basicType);
             test_survey.addAnswers("{\"id\":\"5ec3008f2f8924036eecd3ac\",\"answers\":{\"Is this working?\":\"Yes.\"}}");
             test_survey.addAnswers("{\"id\":\"5ec3008f2f8924036eecd3ac\",\"answers\":{\"Is this working?\":\"Very much so!\"}}");
             surveysRepository.save(test_survey);
+
```

Comment:
maybe split those blocks in `loadSamples` into some methods to better visualise whats going on here and to make it easier to modify samples